pkg/util/rolloutblock/rolloutblock.go

```diff
@@ -0,0 +1,58 @@
+package rolloutblock
+
+import (
+	"github.com/golang/glog"
+	"strings"
+
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+	shippererrors "github.com/bookingcom/shipper/pkg/errors"
+)
+
+// Return a boolean to decide if the application should override one of the
+// existing rollout blocks, a string representing the existing rollout blocks
+// that are not overridden, and an error if the application is trying to override
+// a non-existing rolloutblock
+func ShouldOverrideRolloutBlock(overrideRB string, nsRBs []*shipper.RolloutBlock, gbRBs []*shipper.RolloutBlock) (bool, string, error) {
+	overrideRBs := strings.Split(overrideRB, ",")
+	if len(overrideRB) == 0 {
+		overrideRBs = []string{}
+	}
+	RBs := append(nsRBs, gbRBs...)
```

Comment:
consider calling it `rbs`, as a leading capital-char naturally implies on var public visibility